test/location.coffee

```diff
@@ -595,18 +595,21 @@ test "Verify heregexes with interpolations have the right ending position", ->
   eq comma[0], ','
   eq arrayEnd[0], ']'
 
-  assertColumn = (token, column) ->
+  assertColumn = (token, column, width = 0) ->
     eq token[2].first_line, 0
     eq token[2].first_column, column
     eq token[2].last_line, 0
     eq token[2].last_column, column
+    eq token[2].last_column_exclusive, column + width
 
   arrayEndColumn = arrayEnd[2].first_column
-  for token in [comma, flagsString]
+  for token in [comma]
```

Comment:
This test needed its location data updated to match the refined location data for generated heregex tokens